setup.cfg

```diff
@@ -0,0 +1,57 @@
+[metadata]
+name = aiida-testing
+author = Dominik Gresch
```

Comment:
While we're at it - I'm not the only author of this (anymore). Should at least include

```
Dominik Gresch, Leopold Talirz, Jens Bröder, and the AiiDA team
```

The `author_email` can be left blank I suppose.

Reply:
(you can put yourself also if you want, of course)